### ISSUE TITLE
Eligibility Verification for Seniors without an SSN

### ISSUE THREAD

**Issue by afeld** [cal-itp/benefits#855]
**Date**:
**Writers**:
**Status**: 
**Priority**:  [No. 3 per the roadmap](https://docs.google.com/document/d/1IFoa8Ye0IXwGXXwxFjMrm1s3617Dbv6l-E-aCB0kgnA/edit)

## Background

[Background](https://docs.google.com/document/d/1TtlfJw7HNtyw6Xlcqgg_PHBLc7dCzlZa--cOC2XI2eo/edit#heading=h.bxcxh6h6nikz)

## Decision drivers 

Split out from https://github.com/cal-itp/benefits/issues/635 and its predecessors to unblock the team.

## Decision outcome

We are going to use a new DMV API for verification using a rider's drivers license information.

## Acceptance criteria

- [ ] An elderly adult without a social security number is able to register online for discounted fares when using their bank card

## Definition of Done

A clear and concise description of what "Done" means for this Epic.

## User Stories

- [ ] As a User Story, I want to be verifiable, so that this epic can be satisfied

## Tasks

- [x] Write [proposal approach around the new approach](https://docs.google.com/document/d/1TtlfJw7HNtyw6Xlcqgg_PHBLc7dCzlZa--cOC2XI2eo/edit#)

## Implementation Ready Checklist

- [ ] Acceptance criteria defined
- [ ] Team understands acceptance criteria
- [ ] Acceptance criteria is verifiable
- [ ] User workflows defined / designed
- [ ] External / 3rd Party dependencies identified

## Links, resources

- Other links here

**Comment by thekaveman** [comment]
I think the intention was to re-purpose Epic #363, since that's where most of this work already happened: https://github.com/cal-itp/benefits/issues/363#issuecomment-1147961333

Either way good to have one to track the new work.

**Comment by afeld** [comment]
Since most tasks in #363 are done, I suggest we start with a clean one. That way, each can correspond to a row of the [roadmap](https://docs.google.com/document/d/1IFoa8Ye0IXwGXXwxFjMrm1s3617Dbv6l-E-aCB0kgnA/edit#).

**Comment by thekaveman** [comment]
> That way, each can correspond to a row of the [roadmap](https://docs.google.com/document/d/1IFoa8Ye0IXwGXXwxFjMrm1s3617Dbv6l-E-aCB0kgnA/edit#).

The roadmap previously linked to the existing Epic for the DMV row? Otherwise that Epic doesn't actually correspond to any row in there. I'm not tied to which one we keep open, it just seemed like @vykster wanted to re-purpose the existing one.

**Comment by vykster** [comment]
Not wedded to the old one -- feel free to make a new, just update the link on the roadmap.

**Comment by vykster** [comment]
deprioritizing and closing. Will create new issue/epic if raised as a priority again.
